### PR TITLE
PAD-53

### PR DESCRIPTION
**Pull request by Nikhil-Nunna**
## Description
Updating AWS-OFI and NCCL versions for TF2-GPU-OFI image.

## Checklist

- [ ] Bump VERSION to make the pushed images are tagged with the right version.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.
- [ ] Test the images by running the test `bumpenvs` procedure in the determined repo. See [README](https://github.com/determined-ai/environments/blob/main/README.md).

### CODE REVIEW

**Comment by Nikhil-Nunna** [comment]
All of the containers built correctly however the [publish-cloud-images-dev](https://circleci.com/gh/determined-ai/environments/8919?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary) step is failing.

**Comment by ioga** [comment]
```
1694452919,,ui,message,    amazon-ebs-gov: failed: [default] (item=deb https://nvidia.github.io/libnvidia-container/ubuntu20.04/amd64 /) => {"ansible_loop_var": "item"%!(PACKER_COMMA) "changed": false%!(PACKER_COMMA) "item": "deb https://nvidia.github.io/libnvidia-container/ubuntu20.04/amd64 /"%!(PACKER_COMMA) "msg": "Failed to update apt cache: E:The repository 'https://nvidia.github.io/libnvidia-container/ubuntu20.04/amd64  Release' does not have a Release file."}
```

same operation worked fine on non-gov build. if it happened once I'd suggest rerunning it.

if it's persistent, someone'll have to investigate what's wrong with nvidia docker repos on govcloud.

**Comment by Nikhil-Nunna** [comment]
I already re-ran it once, so I got the same error twice, should I try my luck a 3rd time or stop there?

File: dockerfile_scripts/scrape_libs.sh [hunk 1]
```diff
@@ -41,10 +41,10 @@ then
    export CUDA_CACHE_PATH=${CUDA_CACHE_PATH:=${tmp_nvcache_dir}}
 
    export TF_FORCE_GPU_ALLOW_GROWTH=${TF_FORCE_GPU_ALLOW_GROWTH:=true}
-
    # NOTE: Disable memory registration to workaround the current issues
    #       between libfabric and cuda.  When those issus are resolved,
    #       simply set the vaiable to 0 before launching the container.
+   export FI_MR_CACHE_MONITOR=userfaultfd
```
Comment (karlonw) [inline dockerfile_scripts/scrape_libs.sh:47]: Should we add a comment here on what this variable does/why it's important ?

Comment (Nikhil-Nunna) [reply dockerfile_scripts/scrape_libs.sh:47]: Yeah I can make that change.

Comment (Nikhil-Nunna) [reply dockerfile_scripts/scrape_libs.sh:48]: Comments have been added!

**Pull request closed without merging**